pkgs/development/python-modules/missingno/default.nix

```
@@ -0,0 +1,39 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, matplotlib
+, scipy
+, seaborn
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "missingno";
+  version = "0.4.2";
+
+  src = fetchFromGitHub {
+    owner = "ResidentMario";
+    repo = pname;
+    rev = version;
+    sha256 = "0wxhj7qm2fcaiprdjfrx9p8nikc10qhivf7rzwi04r6mrzyi0i9x";
+  };
+
+  propagatedBuildInputs = [
+    matplotlib
+    scipy
+    seaborn
+  ];
+
+  checkInputs = [ pytest ];
+
+  checkPhase = ''
+  pytest tests/util_tests.py -k 'not cutoff'
```

Indentation + missing hooks, see https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/explicit-phases.md .

```suggestion
    runHook preCheck

    pytest tests/util_tests.py -k 'not cutoff'

    runHook postCheck
```

---

Also pytestCheckHook is prefered.